docs/source/indepth_tutorial/directed-search.ipynb

```diff
@@ -98,9 +98,9 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    limits.loc[0, ["utility", "inertia", "reliability", "max_P"]] = 0
</pre><p>Add a comment that <code>limits.loc[0]</code> correlates with the lower limit, and limits.loc[1] to the upper limit</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quaquel/EMAworkbench/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>